run/jobs/main.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+type EnvVars struct {
+	// Job-defined env vars
+	taskNum    string
+	attemptNum string
+
+	// User-defined env vars
+	sleepMs  int64
+	failRate float64
+}
+
+func stringToInt(s string) int64 {
+	num, _ := strconv.ParseInt(s, 10, 64)
```

Comment:
suggestion: Suppressing the parsing errors is not idiomatic. I suggest making these more specific about the variable they support and adding a log.Fatal, or returning the error and handling it in main(). This comes close to blocker but I think falls just short.

In my opinion, I would have a `configFromEnv() (Config, error)` function and fatal the error in main. Config in this case is a replacement name for `type EnvVars`, what's important about them is not that they are environment, but that they are the config from the app that happens to come from the environment.